Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM node:boron
 
 # Update and install packages
 RUN apt-get update && \
-    apt-get install -y python python-dev python-pip python-virtualenv zip rsync && rm -rf /var/lib/apt/lists/*
+    apt-get install -y python python-dev python-pip python-virtualenv zip rsync default-jdk && \
```

Comment:
`RUN export JAVA_HOME` is not needed.

Instead of `RUN export PATH="$JAVA_HOME/bin:$PATH"` do,

`ENV PATH $JAVA_HOME/bin:$PATH